src/Components/Publishing/Sections/VerticalOrSeriesTitle.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react"
+import styled from "styled-components"
+import { getEditorialHref } from "../Constants"
+import { pMedia } from "../../Helpers"
+import { ArticleData } from "../Typings"
+import { Fonts } from "../Fonts"
+
+interface Props {
+  article?: ArticleData
+  color?: string
+  prefix?: string
+  vertical?: any
+}
+
+export const VerticalOrSeriesTitle: React.SFC<Props> = props => {
```

Comment:
Is there a design name for this? Like a Tagline or something? I definitely don't mind the verboseness of the component but just a thought if this expands to include more possible text types.

Reply:
Thought about this, but decided against because it is used in both the article header and the series footer atm. I'm open to changing, but feel we may want to wait until the opportunity for more uses arises.